### ISSUE TITLE
Unable to run batch madcow by selected file name

### ISSUE THREAD

**Issue by edwardvelasco** [4impact/madcow#42]
Unable to execute batch madcow using the ff. commands below:

1. `runmadcow.bat -e TESTLOCAL -t f_addCard, f_editCard, f_listClients, f_removeClient, f_viewClient`
- Error Message:
```
C:\Edward\Billzy\madcow>runmadcow.bat -e TESTLOCAL -t f_addCard, f_editCard, f_listClients, f_removeClient, f_viewClient
tests: [f_addCard, , --, f_editCard, , f_listClients, , f_removeClient, , f_viewClient]
2016-04-21 09:06:47,258 INFO [au.com.ps4impact.madcow.report.MadcowExecutionReport] - <Preparing assets for Madcow Report...>
There was an error running Madcow: Unable to find matches on the classpath for '**/.grass'
Press any key to continue . . .  
```

2. `runmadcow.bat -e TESTLOCAL -t functional/f_addCard, functional/f_editCard`
- Error Message:
```
C:\Edward\Billzy\madcow>runmadcow.bat -e TESTLOCAL -t functional/f_addCard, functional/f_editCard
tests: [functional/f_addCard, , --, functional/f_editCard]
2016-04-21 09:13:25,994 INFO [au.com.ps4impact.madcow.report.MadcowExecutionReport] - <Preparing assets for Madcow Report...>
There was an error running Madcow: Unable to find matches on the classpath for '**/functional/f_addCard.grass'
Press any key to continue . . .
```

3. `runmadcow.bat -e TESTLOCAL -t tests/functional/f_addCard, tests/functional/f_editCard`
- Error Message:
```
C:\Edward\Billzy\madcow>runmadcow.bat -e TESTLOCAL -t tests/functional/f_addCard, tests/functional/f_editCard
tests: [tests/functional/f_addCard, , --, tests/functional/f_editCard]
2016-04-21 09:13:38,849 INFO [au.com.ps4impact.madcow.report.MadcowExecutionReport] - <Preparing assets for Madcow Report...>
There was an error running Madcow: Unable to find matches on the classpath for '**/tests/functional/f_addCard.grass'
Press any key to continue . . .
```

**Comment by edwardvelasco** [comment]
Retry the command below just removed the space and it works as expected.
1. runmadcow.bat -e TESTLOCAL --test f_addCard.grass,f_editCard.grass
2. runmadcow.bat -e TESTLOCAL --test f_addCard,f_editCard

**Comment by dawogfather** [comment]
This should be fixed in build #125 release tag v2.0.0-alpha.3 as we are now trimming the "test" arguments
